app/models/print_visits.rb

```diff
@@ -1,5 +1,21 @@
 class PrintVisits
   include MemoryModel
+  # include ActiveModel::Validations
 
   attribute :visit_date, :accessible_date
+
+  validate :check_date, on: :create
+
+  def check_date(date)
```

Comment:
no need to pass the date. We have access to `@accessible_date` then you can `#to_date` it